examples/hello-world/main.cpp

```diff
@@ -114,7 +114,7 @@ int HelloWorldExample::createComputePipelineFromShader()
     slang::SessionDesc sessionDesc = {};
     slang::TargetDesc targetDesc = {};
     targetDesc.format = SLANG_SPIRV;
-    targetDesc.profile = slangGlobalSession->findProfile("glsl440");
+    targetDesc.profile = slangGlobalSession->findProfile("glsl_440");
```

Comment:
This example is compiling to Vulkan, so it seems the right profile to use here is `spirv_1_5`.

Reply:
@chaoticbob , can you make a new change based on Yong's suggestion?
It appears that I approved the change sooner than I should have.

Reply:
@jkwak-work of course. Done in pr#4297.